src/controllers/Decks.js

```diff
@@ -31,27 +40,43 @@ const getAllDecks = async (req, res) => {
 
```

Comment:
Could we use more descriptive error messages throughout our code to give user more details as to what goes wrong? I wonder if this is an uodate worth making using our middleware status code file